### DISCUSSION TITLE
Give the girls more money....

### DISCUSSION THREAD

**Discussion by MICNeuromancer** [pH7Software/pH7-Social-Dating-CMS#923] [Ideas/Suggestions]
Hello... how I can protect a image video folder with a password? I like to give all girls on ph7cms more possibilities to make money with there content... 
what I need that's the girls can protect each folder and if someone pay maybe over PayPal the girls can give it free...

Or a new kind of button: pay xxx to see the content from the folder...

Or in the best case functions like patreon...

**Comment by hobb7t** [comment]
Sounds pretty interesting! Let's hope that someone with php knowledge will take the time to make such option available
